src/NUnitFramework/framework/Internal/Filters/CompositeFilter.cs

```diff
@@ -21,7 +21,6 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
-using System;
```

Comment:
Thanks for doing opportunistic cleanup like this too

Reply:
I try to do this only for files I touch, this was a bit on the fence as the addition of `sealed` was not required for the functionality 🤷🏻‍♂️